extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -157,42 +161,24 @@ public static boolean createTableIfNotExists(final Configuration conf, final Str
         return false;
     }
 
-    private static String getIndexTableName(final Configuration conf, final String indexTableNameConf, final String altSuffix){
-        String value = conf.get(indexTableNameConf);
-        if (value == null){
-            final String defaultTableName = conf.get(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX);
-            Validate.notNull(defaultTableName, indexTableNameConf + " not set and " + RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX + " not set.  Cannot generate table name.");
-            value = conf.get(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX) + altSuffix;
-        }
-        return value;
-    }
+    /**
+     * Lookup the table name prefix and throw an error if it is null.
+     * TODO get table prefix from RyaDetails
+     * @param conf
+     * @return
+     */
+	public static String getTablePrefix(final Configuration conf) {
+		final String tablePrefix;
+		tablePrefix = conf.get(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX);
+		Validate.notNull(tablePrefix, RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX + " not set.  Cannot generate table name.");
```

Comment:
You could use Objects.requireNonNull(object, message) here instead of depending on a library.

https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html